api/imagescan.go

```diff
@@ -73,6 +73,16 @@ func ScanImages(c *gin.Context) {
 	if !ok {
 		return
 	}
+
+	if !commonCluster.GetSecurityScan() {
+		c.JSON(http.StatusNotFound, pkgCommmon.ErrorResponse{
```

Comment:
Can't `common.ErrorResponseWithStatus` be used here to send back error response?

Reply:
Thanks for mentioning common.ErrorResponseWithStatus function.

Reply:
thx @pbalogh-sa